### PR TITLE
Add mandatory-claims and mapping-claims validations to status check

### PR DESCRIPTION
**Pull request by nessiLahav**
### What does this PR do?
Add `mandatory-claims` and `mapping-claims` validations to status check

#### Change log
- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

#### API Changes
- [ ] The [OpenAPI spec](https://github.com/cyberark/conjur-openapi-spec) has been updated to meet new API changes (or an issue has been opened), or
- [ ] The changes in this PR do not affect the Conjur API

### CODE REVIEW

File: app/domain/authentication/authn_jwt/validate_status.rb [hunk 1]
```diff
@@ -79,7 +83,17 @@ def validate_webservice_is_whitelisted
 
       def validate_issuer
         @fetch_issuer_value.call(authentication_parameters: authentication_parameters)
-        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedIssuerConfiguration.new)
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedMandatoryClaimsConfiguration.new)
```
Comment (sashaCher) [inline app/domain/authentication/authn_jwt/validate_status.rb:86]: ???

File: app/domain/logs.rb [hunk 2]
```diff
@@ -658,6 +658,16 @@ module AuthnJwt
         msg: "Successfully fetched mapping claims '{0-mapping-claims}'",
         code: "CONJ00130I"
       )
+
```
Comment (nessiLahav) [inline app/domain/logs.rb:661]: @hilagross @shulifink Please review

**Review by sashaCher — DISMISSED** [review]
LGTM

**Review by sashaCher — APPROVED** [review]
LGTM

**Review by tzheleznyak — APPROVED** [review]
LGTM

**Pull request merged**